src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,71 @@
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[] | null;
+  selectedPerson?: string | null;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, selectedPerson }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people?.map(person => {
+          const { sex, born, died, fatherName, motherName, slug } = person;
+          const personMother = people.find(
+            parent => parent.name === motherName,
+          );
+          const personFather = people.find(
+            parent => parent.name === fatherName,
+          );
```

Comment:
Consider optimizing the lookup for `personMother` and `personFather`. Currently, `people.find()` is called twice for each person, which can be inefficient if the list is large. You might consider creating a map of names to people outside the map function to reduce the number of iterations.